dome.gd

```diff
@@ -80,3 +99,7 @@ func _on_dome_lost_countdown_timer_timeout():
 	infestation_stage = InfestationStage.LOST
 	$DomeStatus.text = "Lost"
 	$Building/BuildingSprite.modulate = Color.DIM_GRAY
+
+func _on_resource_generation_timer_timeout():
+	if resource_type != Globals.ResourceType.RESEARCH:
+		generate_resource()
```

Comment:
are you not supposed to generate research?

Reply:
Research is to be generated by scientists around growing infestation, rate of research gen is determined by # research domes uninfested (i.e. labs that samples can be processed by)

Reply:
This will be implemented in #23

Reply:
ah I see 👍